geode-core/src/main/java/org/apache/geode/management/internal/configuration/realizers/PdxRealizer.java

```diff
@@ -25,12 +25,12 @@
   public PdxInfo get(Pdx config, InternalCache cache) {
     PdxInfo info = new PdxInfo();
     info.setReadSerialized(cache.getPdxReadSerialized());
-    info.setDiskStore(cache.getPdxDiskStore());
+    info.setDiskStoreName(cache.getPdxDiskStore());
     info.setIgnoreUnreadFields(cache.getPdxIgnoreUnreadFields());
     info.setPersistent(cache.getPdxPersistent());
     PdxSerializer pdxSerializer = cache.getPdxSerializer();
     if (pdxSerializer != null) {
-      info.setSerializerClassName(pdxSerializer.getClass().getName());
+      info.setPdxSerializer(pdxSerializer.getClass().getName());
```

Comment:
If "pdxSerializer" is an instanceof ReflectionBasedAutoSerializer then it should show up in PdxInfo as an AutoSerializer. You can get the patterns and portable values by calling "getConfig()" on the ReflectionBasedAutoSerializer.
The other thing that is missing here is the initParameters when it is not a ReflectionBasedAutoSerializer. You can not get these given a PdxSerializer. But they are easy to get on the locator from the cluster config. On the locator are you actually showing the "Pdx" instance under configuration and "PdxInfo" is only used on non-locators? The bad thing about not showing the initParameters is that it might be the one thing that changed that has not yet taken effect on a server but you would not be able to tell. Would it be possible for  this command to display something about each member whose pdx config is outdated and needs a restart?

Reply:
Since the cluster config pdx will be show in the result under configuration, I'm okay with the PdxInfo have less information.